code/drasil-code/lib/Language/Drasil/Code/Imperative/GOOL/ClassInterface.hs

```diff
@@ -54,6 +54,10 @@ type ExamplePurpose = String
 type ExampleDescr = String
 -- | File contributors
 type Contributor = String
+-- | Input File
```

Comment:
You should document, here, why there may not be an `Input File` (and same with output)